src/iterative_sorting/iterative_sorting.py

```diff
@@ -16,12 +16,24 @@ def selection_sort( arr ):
 
     return arr
 
+print(selection_sort([8,7,10,19,2,22,24,3]))
+
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
-
+    swaper = False
+    while swaper == False:
+        for i in range(0, len(arr) - 1):
+            bubble1 = arr[i]
+            bubble2 = arr[i+1]
+            if bubble1 > bubble2:
+                arr[i] = bubble2
+                arr[i+1] = bubble1
+                swaper = True
     return arr
 
+print(bubble_sort([8,7,10,19,2,22,24,3]))
+
 
```

Comment:
quick note, if you have a range that you know starts at zero, you dont need that first argument, you can just do range(len(arr)-1)